lib/src/pointer_listener.dart

```diff
@@ -60,7 +60,6 @@ class _PointerListenerState extends State<PointerListener> {
                   onEnter: (details) {
                     onPointerEnter();
                   },
-                  hitTestBehavior: HitTestBehavior.translucent,
```

Comment:
The translucent hit test behavior is needed for touch events to reach the Motion widget's child. Removing this would change the value to `opaque`, which would block user interaction with the content. Unless there is another reason for this change, please remove it from this PR. Thank you !